docs/dashboards/synchronize-components.md

```diff
@@ -0,0 +1,102 @@
+Synchronizing components
+===
+
+Components might need to work together to accomplish a specific task.
```

Comment:
> might need to work together to accomplish a specific task.

I'm not sure if that is the best set of words to describe sync. For me, it's sth extra that helps visualize, navigate and highlight some of the data.